inverse_covariance/tests/two_way_standard_scaler_test.py

```diff
@@ -0,0 +1,207 @@
+import pytest
+import numpy as np
+from scipy import sparse
+from sklearn.utils.testing import assert_raises, assert_allclose
+from sklearn.exceptions import NotFittedError
+
+from inverse_covariance.two_way_standard_scaler import (
+    two_way_standardize,
+    TwoWayStandardScaler,
+)
+
+
+def custom_init(n_rows, n_cols, with_mean=False):
+    prng = np.random.RandomState(1)
+    X = prng.normal(0, np.ones(shape=(n_rows, n_cols)))
+    if with_mean:
+        mu = np.ones(shape=(n_rows, 1)) * prng.randint(1, 5, size=(1, n_cols))
+    else:
+        mu = np.zeros(shape=(n_rows, n_cols))
+
+    var_rows = prng.lognormal(2, 1, size=(n_rows, 1))
+    sqcov_rows = np.diag(np.sqrt(var_rows))
+    var_cols = prng.lognormal(2, 1, size=(1, n_cols))
+    sqcov_cols = np.diag(np.sqrt(var_cols))
+
+    return mu + sqcov_rows * X * sqcov_cols
+
+
```

Comment:
@mnarayan please review all of the tests in the file to see if its behaving as you would expect.  You can add more tests to the parameterize dectorator if you know of good specific small examples.